region/maridia/inner-yellow.json

```diff
@@ -1084,30 +1414,58 @@
           "to": [
             {
               "id": 1,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
               "id": 3,
-              "requires": [
-                {"or": [
-                  "canUseFrozenEnemies",
-                  "SpaceJump",
-                  "canTrickyWallJump",
-                  {"enemyKill":{
-                    "enemies": [
-                      [
-                        "Ripper", "Ripper", "Ripper"
+              "strats": [
+                {
+                  "name": "Frozen Rippers",
+                  "notable": false,
+                  "requires": ["canUseFrozenEnemies"]
+                },
+                {
+                  "name": "Space Jump",
```

Comment:
Is this notably easier than tricky walljumps?

Reply:
I'm honestly not sure. To be it's noticeably more difficult. You think this strat should just be removed?

Reply:
yeah, I'd remove it.  The walljumps are the same level of 'tricky' as the spacejumps, navigable just like the top of red tower.